README.md

```diff
@@ -51,6 +51,21 @@ Disclaimer: may not cover all your Zanata-building needs.
 
 The `-h` argument prints the build script's help.
 
+#### Building on Mac OS X and macOS Sierra
+
+Currently, there is an extra step needed to build Zanata on a Mac.
+For `./build` to work, you will need to point to the correct Java directory using 
+the following command (using the correct JDK version on your Mac):  
+
+`export JAVA_HOME="/Library/Java/JavaVirtualMachines/jdk1.X.X_XX
```

Comment:
Just for clarity, I would suggest changing jdk1.X to jdk1.8, since it must be at least Java 1.8 (and we haven't tried using the 1.9 pre-release).